setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
I'll figure it out and send new PR to address the `install` issue since I understand your use case now.

Though, frankly, I'd say that if you're installing from GitHub, should always use `-e` IMO. ;)

But I'll see what I can do.